vic/drivers/image/src/vic_init_output.c

```diff
@@ -160,12 +175,16 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining nlayer dimenension in %s", nc->fname);
     }
 
-    status = nc_def_dim(nc->nc_id, "ni", nc->ni_size, &(nc->ni_dimid));
+    status =
+        nc_def_dim(nc->nc_id, options.DOMAIN_LON_VAR, nc->ni_size,
+                   &(nc->ni_dimid));
```

Comment:
I think we need to do a conditional here for the number or coordinate dims
1) if `coord_dims == 1`, then the dim names are `lon/lat`
2) if `coord_dims == 2`, then the dim names are `ni/nj`